docs/how-to/configure.md

```diff
@@ -73,21 +73,15 @@ Values for the *Custom* preset can be entered as values, or changed with the arr
 
 The *Telemetry* switch enables gathering and sending basic **anonymous** data to Percona, which helps us to determine where to focus the development and what is the uptake for each release of PMM. Specifically, gathering this information helps determine if we need to release patches to legacy versions beyond support, determining when supporting a particular version is no longer necessary, and even understanding how the frequency of release encourages or deters adoption.
 
-The following information is gathered:
-
-- PMM Server Integration Alerting feature enabled/disabled
-- PMM Server Security Thread Tool feature enabled/disabled
-- PMM Server Backup feature enabled/disabled
-- PMM Server DBaaS feature enabled/disabled
-- PMM Server Check Updates feature disabled
-- Detailed information about the version of monitored MySQL services
-- Monitored MongoDB services version
-- Monitored PostgreSQL services version
-- Total Grafana users
-- Monitored nodes count
-- Monitored services count
-- Agents version
-- Node type
+PMM collects data through exporters and Query Analytics, which is then utilized by the Telemetry subsystem through aggregated metrics. 
```

Comment:
```suggestion
PMM Telemetry is based on data collected by various PMM components and stored inside PMM server 
```
no need to mention components as they might change. and actually, we use not only exporters and qan but some grafana and UI data.